topi/python/topi/generic/nn.py

```diff
@@ -212,3 +211,21 @@ def schedule_binary_dense(outs):
         The computation schedule for the op.
     """
     return _default_schedule(outs, False)
+
+
+@tvm.target.generic_func
+def schedule_upsampling(outs):
```

Comment:
since it uses the default injective schedule, we should simply remove schedule_upsampling function and call schedule_injective instead.